test/utils.py

```diff
@@ -529,7 +529,8 @@ def assert_dependencies_deb(filename, deps, tarantool_versions, tmpdir):
 
     rc, output = run_command_and_get_output(cmd, cwd=tmpdir)
     assert rc == 0
-    assert all(dep in output for dep in deps)
+    # Remove hash from dependencies
+    assert all(dep.split('-')[0] in output for dep in deps)
```

Comment:
See https://github.com/tarantool/cartridge-cli/pull/629/commits/ba99b8de9437e39d6c9d7f65e1139b7aed8311aa#r727489690. I think we should not spead the logic of costructing the minimal version across the code so much.